### PR TITLE
Bug/EUI 3466 date fallback

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3466
https://tools.hmcts.net/jira/browse/EUI-3497

### Change description ###
MO unassigned cases does not show all unassigned cases

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by kuda1992 — DISMISSED** [review]
looks good

File: src/shared/components/case-list/case-list.component.ts [hunk 1]
```diff
@@ -43,11 +43,11 @@ export class CaseListComponent {
   constructor(private browserService: BrowserService) { }
 
   public formatDate(date: Date): string {
-    return formatDate(date, 'dd MMM yyyy', 'en-GB');
+    return date ? formatDate(date, 'dd MMM yyyy', 'en-GB') : '-';
   }
 
   public formatDateAtTime(date: Date): string {
-      return DateTimeFormatUtils.formatDateAtTime(date, false);
+    return date ? DateTimeFormatUtils.formatDateAtTime(date, false) : '-';
```
Comment (paul-graham) [inline src/shared/components/case-list/case-list.component.ts:50]: I don't think the unit tests are actually testing this one at all. Can you add one for it, please?

**Review by paul-graham — CHANGES_REQUESTED** [review]
I think we're missing a unit test (or two).

File: src/shared/components/case-list/case-list.component.ts (hunk 1 above)
Comment (RiteshHMCTS) [reply src/shared/components/case-list/case-list.component.ts:50]: Added unit test case for formatDateAtTime function

**Review by paul-graham — APPROVED** [review]
Looks like there's now a test in place for the date-time formatter too. Thanks, @RiteshHMCTS. 👍

**Review by kuda1992 — DISMISSED** [review]
look good

**Review by kuda1992 — DISMISSED** [review]

**Review by paul-graham — APPROVED** [review]

**Review by kuda1992 — APPROVED** [review]

**Review by kuda1992 — APPROVED** [review]
looks good

**Review by kuda1992 — APPROVED** [review]

**Review by paul-graham — APPROVED** [review]

**Pull request merged**